liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -952,6 +952,7 @@ private void writeJvmOptions(File file, List<String> options, List<String> maven
                 combinedJvmOptions = new ArrayList<String> ();
                 // add the maven properties first so that they do not take precedence over the options specified with jvmOptions
                 combinedJvmOptions.addAll(mavenProperties);
+                combinedJvmOptions.removeAll(options); // remove any exact duplicates before adding all the jvmOptions
```

Comment:
OK, I see.  This is avoiding dup values across <jvmOptions> parms and the properties.   In the process of understanding this, it also occurred to me that there's still the possibility of dups among the <jvmOptions> entries.   You'd probably need to imagine pluginManagement with inheritance for this ever happening.. and we are getting into a real corner case..but I thought of it so mentioning it.

Reply:
For the properties themselves, I believe I have removed the possibility of duplicates with this change:

```
case JVM:        if (!jvmMavenProps.contains(value)) { jvmMavenProps.add(value); } // avoid exact duplicates
```

Then the change you made the comment on here ensures we do not include duplicates between the jvmOptions config and the Maven properties (both in pom.xml and command line). So I actually don't think we can get duplicates entries now.

Reply:
The thing that's left is within `<jvmOptions>` parameter config elements.   Yeah, it seems far-fetched to think someone would duplicate a value in a single file.   

Maybe something like this is more in the realm of realism:

### parent

```xml
<pluginManagement>
  ... 
    <jvmOptions>
      <parm>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=7777</parm>
    </jvmOptions>
```

### child

```xml
  ... 
    <jvmOptions>
      <parameter>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=7777</parameter>
    </jvmOptions>
```

Since I think you can use any element name here to "wrap" the JVM option value.   

And again, I wouldn't have even thought of this until I understood what the code you added was doing.